app/Notifications/ReviewNotes.php

```diff
@@ -52,12 +52,16 @@ public function via($notifiable)
     public function toMail($notifiable)
     {
         if ($notifiable !== null && $notifiable->invites == 1) {
+            $locale = $notifiable->language;
+
             return (new MailMessage)
                   ->subject('A repair has been marked as suitable for the wiki')
```

Comment:
A couple of lines here don't seem to be going through translations.

Reply:
ReviewNotes is only sent to admins, though, right?  The aim for this story is only to have done all the non-admin ones.

Reply:
Oh yes, that's fine then.

Reply:
Actually it seems to have been removed in another branch; I think James decided he didn't need it.